tests/integration/multi_table/test_hma.py

```diff
@@ -1788,3 +1790,62 @@ def test_detect_from_dataframe_numerical_col():
 
     test_metadata = MultiTableMetadata()
     test_metadata.detect_from_dataframes(data)
+
+
+def test_table_name_logging(caplog):
+    """Test the table name is correctly logged GH#1964."""
+    # Setup
+    parent_data = pd.DataFrame({
+        'parent_id': [1, 2, 3, 4, 5, 6],
+        'col': ['a', 'b', 'a', 'b', 'a', 'b'],
+    })
+    child_data = pd.DataFrame({
+        'id': [1, 2, 3, 4, 5, 6],
+        'parent_id': [1, 2, 3, 4, 5, 6]
+    })
+    data = {
+        'parent_data': parent_data,
+        'child_data': child_data,
+    }
+    metadata = MultiTableMetadata()
+    metadata.detect_from_dataframes(data)
+    instance = HMASynthesizer(metadata)
+
+    # Run
+    with catch_sdv_logs(caplog, logging.INFO, 'sdv.data_processing.data_processor'):
+        instance.fit(data)
+
+    # Assert
+    assert caplog.messages == [
+        'Fitting table parent_data metadata',
```

Comment:
Wouldn't it be better to loop through the messages and check that the substring `table parent_data metadata` is present in each message?